modules/telariaBidAdapter.js

```diff
@@ -112,21 +112,22 @@ export const spec = {
  * Generates the url based on the parameters given. Sizes, supplyCode & adCode are required.
  * The format is: [L,W] or [[L1,W1],...]
  * @param bid
+ * @param bidderRequest
  * @returns {string}
  */
-function generateUrl(bid) {
+function generateUrl(bid, bidderRequest) {
   let width, height;
-  if (!bid.sizes) {
+  let playerSize = (bid.mediaTypes && bid.mediaTypes.video && bid.mediaTypes.video.playerSize);
+  if (!playerSize) {
     return '';
```

Comment:
the documentation states that playerSize is an optional field -- do you want to consider a default here?

Reply:
I've removed the changes to the package-lock.json files as well. Please let me know if there's anything else.